R/skip-vcr-off.R

```diff
@@ -0,0 +1,27 @@
+#' Skip tests if vcr is off
```

Comment:
i'd prefer to keep lines (docs and code) 80 characters or narrower - referring to all lines here